services/shared/matrix/__init__.py

```
@@ -0,0 +1,43 @@
+    "BundleKind",
+    "MediaDeletion",
+    "ProvisionResult",
+    "ProvisioningError",
+    "PurgeHandle",
+    "PurgeStatus",
+    "SyncBundle",
+    "SynapseAdminClient",
+    "SynapseProvisioner",
+    "UserActivity",
+    "UserMediaUsage",
+]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)